wait/awaitility.go

```diff
@@ -14,15 +17,13 @@ import (
 	"sigs.k8s.io/kubefed/pkg/apis/core/common"
 	"sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
 	"sigs.k8s.io/kubefed/pkg/controller/util"
-	"testing"
-	"time"
 )
 
 const (
 	OperatorRetryInterval          = time.Second * 5
 	OperatorTimeout                = time.Second * 60
-	RetryInterval                  = time.Millisecond * 100
-	Timeout                        = time.Second * 3
+	RetryInterval                  = time.Millisecond * 500
+	Timeout                        = time.Second * 10
```

Comment:
This is done mainly for delete_namespace.

More timeout is fine for positive case (it will not have any impact).
More timeout only affect negative case and waiting more should be fine.

I spend some time to adjust a bit to have same interval/timeout to avoid complexity in code .. Done https://github.com/codeready-toolchain/toolchain-e2e/pull/5/commits/1b9e6f17f71c059e419053731a5073d9287ff540.  In case need be, I am fine changing to diff timeout/interval.